public/app/core/components/AppChrome/ReportIssueButton/ReportIssueButton.tsx

```diff
@@ -247,6 +254,17 @@ const DrawerContents = ({
       });
   };
 
+  const [accessChecked, setAccessChecked] = useState(false);
```

Comment:
rather than have the state live in this component you could put these as features in the formData obj that comes in as props and update them with setFormData func that also comes in as props, then the data will persist

But also if you want me to fix I can!